packages/agoric-cli/src/commands/ec.js

```diff
@@ -207,6 +207,32 @@ export const makeEconomicCommiteeCommand = (_logger, io = {}) => {
       });
     });
 
+  ec.command('find-continuing-id')
```

Comment:
This looks like a copy of `find-continuing-ids`; it's not straightforward to add the `--for` option there?

Reply:
It changes the return type. The end user doesn't care about the implementation similarity